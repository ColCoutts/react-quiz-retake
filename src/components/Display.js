import React from 'react';
import PropTypes from 'prop-types';

function Display({ text, bottomText, image }) {
  const styles = {
    width: '100px',
    height: '100px'
  };

  return (
    <div style={styles}>
      <p>{text}</p>
      <img src={image}></img>
      <p>{bottomText}</p>
    </div>
  );
}

Display.propTypes = {
  text: PropTypes.string.isRequired,
  bottomText: PropTypes.string.isRequired,
  image: PropTypes.string.isRequired
};

export default Display;
