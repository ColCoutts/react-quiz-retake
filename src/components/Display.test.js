import React from 'react';
import { shallow } from 'enzyme';
import Display from './Display';

describe('text selector snapshot', () => {
  it('renders textselector', () => {
    const text = 'hello';
    const bottomText = 'there';
    const image = '/image';
    const wrapper = shallow(<Display text={text} bottomText={bottomText} image={image} />);
    expect(wrapper).toMatchSnapshot();
  });
});
