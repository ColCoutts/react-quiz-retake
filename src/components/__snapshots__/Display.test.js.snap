// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`text selector snapshot renders textselector 1`] = `
<div
  style={
    Object {
      "height": "100px",
      "width": "100px",
    }
  }
>
  <p>
    hello
  </p>
  <img
    src="/image"
  />
  <p>
    there
  </p>
</div>
`;
